src/server.ts

```diff
@@ -56,7 +54,6 @@ export class Server {
   }
 
   private onListening (server: any) {
-    // console.log(server)
     let addr = server.address()
     let bind = typeof addr === 'string'
       ? 'pipe ' + addr
```

Comment:
i suggest a if instead of a too long one-line-if

Reply:
and set the types

Reply:
Done.